stable/prometheus-rabbitmq-exporter/values.yaml

```diff
@@ -47,4 +47,4 @@ rabbitmq:
 annotations: {}
 #  prometheus.io/scrape: "true"
 #  prometheus.io/path: "/metrics"
-#  prometheus.io/port: 9419
+#  prometheus.io/port: "9419"
```

Comment:
Can you change this to the named port `publish`?

Reply:
Not sure I understand what you mean here, I added quote to the annotation because without the quote the deployment file is not valid

Reply:
The change you did is correct. I was suggesting to use the named port called `publish` in the deployment as it's less brittle, but it occurred to me not everyone's scrape_config may support using named ports. So nevermind.